src/coreclr/clrdefinitions.cmake

```diff
@@ -161,6 +161,11 @@ if(FEATURE_OBJCMARSHAL)
   add_compile_definitions(FEATURE_OBJCMARSHAL)
 endif()
 
+if(FEATURE_JAVAMARSHAL)
+  add_compile_definitions(FEATURE_JAVAMARSHAL)
+  add_compile_definitions(FEATURE_GCBRIDGE)
```

Comment:
Why do we need two different FEATURE_... ifdefs for this?

Reply:
At this point there aren't any. While I was implementing it the logic was clearly distinct though. There is the general concept of the GC Bridge and also the specific Java API and infrastructure that exposes that data. It was natural to separate the two given how we've implemented other interop scenarios involving the GC. It is entirely reasonable to collapse them, but making them distinct now is much easier than retroactively splitting them in the future.